### PR TITLE
fix: explore Works Link on Inbox

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [] 

### Description

This PR fixes the issue with the Explore Works link that was reported here in [Slack](https://artsy.slack.com/archives/C02BAQ5K7/p1732213594705919)

https://github.com/user-attachments/assets/71e036fd-b155-432c-895f-815de893b765

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- fix explore works link on Inbox - mounir

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (fix explore works link on Inbox - mounir - MounirDhahri)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 105b10f3329e8b9c1d277573addee8d01b1f1bd8
</p>

File: src/app/Scenes/Inbox/Components/Conversations/NoMessages.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-import { Flex, Text, Button } from "@artsy/palette-mobile"
-import { navigate } from "app/system/navigation/navigate"
+import { Button, Flex, Text } from "@artsy/palette-mobile"
+import { switchTab } from "app/system/navigation/navigate"
 import { useTracking } from "react-tracking"
 
 export const NoMessages: React.FC = () => {
   const tracking = useTracking()
-  const noMessagesButtonText = " Explore works"
+  const noMessagesButtonText = "Explore works"
```
Comment (olerichter00) [inline src/app/Scenes/Inbox/Components/Conversations/NoMessages.tsx:8]:
```suggestion
  const noMessagesButtonText = "Explore Works"
```

**Review by olerichter00 — DISMISSED** [review]
🙏

File: src/app/Scenes/Inbox/Components/Conversations/NoMessages.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/app/Scenes/Inbox/Components/Conversations/NoMessages.tsx:8]: nice idea - I will update it here and in the test

**Pull request merged**